### PR TITLE
Add ability to disable http keep-alives when connecting to Icinga2 API

### PR DESCRIPTION
**Pull request by mc-meta**
Hello,

this PR is the second of two related to:

         https://github.com/vshn/signalilo/issues/29

and is exposing the possibility to tune http keepalive.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -158,11 +160,12 @@ func NewMockConfiguration(verbosity int) Configuration {
...
-			URL:         "localhost:5665",
-			User:        "sepp",
-			Password:    "sepp1",
-			InsecureTLS: true,
-			Debug:       false,
+			URL:               "localhost:5665",
+			User:              "sepp",
+			Password:          "sepp1",
+			InsecureTLS:       true,
+                        DisableKeepAlives: false,
```
Comment (simu) [inline config/config.go:167]: Please use tabs to indent Go code. This is the standard which is applied by [gofmt](https://golang.org/cmd/gofmt/)

**Comment by mc-meta** [comment]
Indentation changes applied. HTH

**Comment by mc-meta** [comment]
Rebase completed.

**Comment by simu** [comment]
Thanks for the contribution :+1:

**Pull request merged**